includes/class-wc-order-refund.php

```diff
@@ -15,11 +15,18 @@
 class WC_Order_Refund extends WC_Abstract_Order {
 
 	/**
+	 * Which data store to load.
+	 *
+	 * @var string
+	 */
+	protected $data_store_name = 'order-refund';
+
+	/**
 	 * Extend the abstract _data properties and then read the order object.
+	 *
 	 * @param int|object|WC_Order $read Order to init.
 	 */
 	 public function __construct( $read = 0 ) {
-		// Extend order data
 		$this->data = array_merge( $this->data, array(
```

Comment:
To match the other classes, the extra fields should be part of `extra_data`, and then merged in.